guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -42,21 +42,31 @@ For information on backups, see {AdministeringDocURL}Backing_Up_Server_and_Proxy
 
 ifdef::katello,satellite[]
 +
-. Regenerate certificates.
-On the main {Project} server:
+. Regenerate certificates on your {ProjectServer}:
+.. Regenerate certificates for {SmartProxies} that use default certificates:
 +
 [options="nowrap" subs="attributes"]
 ----
-# {certs-generate} --foreman-proxy-fqdn "myproxy.example.com" \
-                       --certs-update-all \
-                       --certs-tar "~/myproxy.example.com-certs.tar"
+# {certs-generate} --foreman-proxy-fqdn "{smartproxy-example-com}" \
+    --certs-update-all \
+    --certs-tar "~/{smartproxy-example-com}-certs.tar"
 ----
+.. Regenerate certificates for {SmartProxies} that use custom certificates:
 +
+[options="nowrap" subs="attributes"]
+----
+# {certs-generate} --foreman-proxy-fqdn "{smartproxy-example-com}" \
+    --certs-tar "~/{smartproxy-example-com}-certs.tar" \
+    --server-cert "/root/capsule_cert/capsule_cert.pem" \
```

Comment:
Would you mind renaming "capsule_cert" to something suitable for all build targets? I'm thinking "proxy" in contrast to "server". Or maybe just reuse the attribute?

Reply:
@maximiliankolb I replaced `capsule_cert` with `{certs-proxy-context}_cert`. While doing this, I noticed two issues:
* orcharhino does not have `certs-proxy-context` attribute. Should we add one?
* orcharhino has the value of `smart-proxy-context` attribute set to `orcharhino Proxy`. I assume this is wrong for two reasons:
** Attribute `SmartProxy` has the exact same value for orcharhino
** `smart-proxy-context` is being used in anchors throughout the entire documentation

I would like to open another PR to change the value of `smart-proxy-context` for orcharhino, please let me know what should I change it to.

Reply:
Thanks for spotting this! I think both of your suggestions to add the attribute and reword the other are good. If you add this in a separate commit to your branch, I'd be very happy. Else, I can of course provide a PR doing this later myself. Does this currently block your work? If so, ping me & I'll create a PR right away.

When merging, we can select "rebase and merge" to not squash your commits into one.

Reply:
I added the commit with updating the attributes but I need your help with what to rename them to. I set both `smart-proxy-context` and `certs-proxy-context` to `orcharhino-proxy`. Should I leave them I like this or rename to something else?